<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxCalibr">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Калібровка каналів</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,10">
      <item>
       <widget class="QLabel" name="labelCalibrChannel">
        <property name="text">
         <string>Канал</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCalibrChannels"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,10">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCalibr">
        <property name="text">
         <string>Калібрувати</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonSaveConfig">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Save.ico</normaloff>:/icons/Save.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancelConfig">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Вихід</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/exit.ico</normaloff>:/icons/exit.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxComPort">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>COM Port</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QComboBox" name="comboBoxComPort"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonCalibrMode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>211</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>Режим калібрування печі</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/measurement.ico</normaloff>:/icons/measurement.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
